### DISCUSSION TITLE
airflow install offline in linux redhat7.8

### DISCUSSION THREAD

**Discussion by sonnguyengoc9** [apache/airflow#33726] [Q&A]
when I setup apache-airflow==2.7.0  offline in in linux-redhat7.8, 
I got an "**_ERROR: google_re2-1.1-1-cp38-cp38-manylinux_2_24_x86_64.manylinux_2_28_x86_64.whl is not a supported wheel on this platform._**"?
can some body help me?

**Comment by potiuk** [comment]
I suggest upgrading Redhat to a newer version, it's also possible you have 32 bit platform as one of the problems.

But you have not mentioned how you get your wheels to make your offline installation. It's quite likely that if you do it properly, even if there is no good whl ready for your platform it will be automatically compiled and built for you.

Very likely you downloaded your .whl on a different platform that you install it on. And this is very wrong.

Generally if you run `pip download` to get  .whls for offline installations (I assume that's what you did) - you MUST download them on the same architecture you are installing it later on.  So make sure when you prepare your offline installation, that your architecture, OS etc. are exactly the same as the one you install it on.

Which leads back to the original advice - maybe the right solution is to get whatever you are installing airflow on upgraded to the same platform/OS as you use to download the .whl files on.
